### ISSUE TITLE
Patch option in Sensor

### ISSUE THREAD

**Issue by tvalasek** [argoproj/argo-events#594]
**Is your feature request related to a problem? Please describe.**
Unable to submit Argo Workflow that use WorkflowTemplates. Sensor should be able to re-apply (patch) existing K8s resource. If it doesn't exist then it will create else it will patch.

**Describe the solution you'd like**
Submitting Argo Workflow that use WorkflowTemplates requires WorkflowTemplates to be in K8s cluster prior Argo Workflow kicks off. 

In our case we have a workflow that relies on an existing workflowtemplate existing on the server, we would like to avoid having to perform the `argo template create workflowtemplatefile.yaml` step manually and rather, have this be automatically submitted by the sensor.

Add `patch` option to https://github.com/argoproj/argo-events/blob/019628a9ee914de015dd2cf2974463236f775a16/sensors/triggers/standard-k8s/starndard-k8s.go#L122-L134

**Describe alternatives you've considered**
none

**Additional context**
Slack conversation: https://argoproj.slack.com/archives/CAM37QYMS/p1586884511152200

**Comment by automagic** [comment]
@VaibhavPage @tvalasek  I am wondering, does this also mean that we cannot trigger a workflow from an existing WorkflowTemplate already applied in k8s at this time?   We deploy (apply) our WorkflowTemplates separately via Helm/ArgoCD, but would like to invoke those as a workflow via an event.    Currently we use a ConfigMap containing a Workflow and reference that from Argo Event trigger.... would like to just use a WorkflowTemplate

**Comment by VaibhavPage** [comment]
@automagic you should be able to use `WorkflowTemplate` as a trigger in the sensor. 

something like,

```
        k8s:
          group: argoproj.io
          version: v1alpha1
          resource: workflowtemplates
          operation: create
```

**Comment by VaibhavPage** [comment]
feature is now available in v0.15.0
